yt/frontends/ramses/data_structures.py

```diff
@@ -184,21 +184,15 @@ def __init__(self, ds, domain_id):
         self.ds = ds
         self.domain_id = domain_id
 
-        num = os.path.basename(ds.parameter_filename).split(".")[0].split("_")[1]
-        rootdir = ds.root_folder
-        basedir = os.path.abspath(os.path.dirname(ds.parameter_filename))
-        basename = "%s/%%s_%s.out%05i" % (basedir, num, domain_id)
-        part_file_descriptor = f"{basedir}/part_file_descriptor.txt"
+        num = ds.basename.split(".")[0].split("_")[1]
+        basename = os.path.join(ds.directory, f"%s_{num}.out{domain_id:05}")
```

Comment:
I'm being 100% backward compatible here, but I'm not completely sure if the leftover `%s` is intentional.

Reply:
It is - it is formatted later with a % to match files named e.g. `output_00123/hydro_XXXXX.outYYYYY`, `part_XXXXX.outYYYYY`, etc.

Reply:
thanks !